e2e/ansible/roles/k8s-gke/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Get the details of the available nodes
+  shell: kubectl get nodes | grep gke | awk {'print $1'}
+  args:
+    executable: /bin/bash
+  register: result_node
+  delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+
+- name: Creating random name for the disks
```

Comment:
@dargasudarshan i used the node name also  for the disk. The reason why I'm using random name is, if the disk is not deleted or the disk name is already there  then it will not create new disk. So only i added some random name to the disk attached to each node.

Reply:
Ok. Good